client/blocks/product-purchase-features-list/business-onboarding.jsx

```diff
@@ -23,7 +23,7 @@ export default localize( ( { isWpcomPlan, translate, link, onClick = noop } ) =>
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon={ <img alt="" src={ conciergeImage } /> }
-				title={ translate( 'Concierge orientation' ) }
+				title={ translate( 'Quick Start session' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Checkout ‹ Quick Start Session' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).